hugo/layouts/why-sourced/single.html

```diff
@@ -0,0 +1,90 @@
+            </div>
+            <div class="col-sm-6">
+                <p>Lack of visibility comes from:</p>
+                <ul class="small-list">
+                    <li>Volume, variety, intricacy and versioning of source code</li>
+                    <li>Source code, software development and business data are spread across many silos</li>
+                    <li>Large volume, variety, intricacy and versions of data</li>
+                    <li>Retrieving, storing and querying at scale is hard</li>
+                    <li>Shadow IT and “Dark Data” due to developer frustration with legacy systems & slow processes</li>
+                </ul>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section id="success">
+    <div class="container text-dark">
+        <header class="text-center my-5">
+            <h2>source{d} gives IT executives visibility into codebases, IT teams & processes</h2>
+        </header>
+        <div class="row">
+            <div class="col-sm-6">
+                <h3 class="title-line line-yellow">Codebases</h3>
+                <p class="mb-5">Make wiser use of existing technology, reduce duplication of software assets. Accelerate your IT modernization and Cloud Migration.</p>
+                <h3 class="title-line line-yellow">Teams</h3>
+                <p class="mb-5">Quickly assess your talent pool, knowledge gaps and effectively manage your teams with data-driven KPIs.</p>
+                <h3 class="title-line line-yellow">Processes</h3>
+                <p class="mb-5">Identify patterns, drive innovation and collaboration in your organization.</p>
+            </div>
+            <div class="col-sm-6">
+                <img class="img-fluid" src="img/visibility_panels.jpg" alt="">
+            </div>
+        </div>
+    </div>
+</section>
+
+<section id="journey" class="journey text-white pb-5">
+    <div class="container text-center pt-5">
+        <header class="mb-5">
+            <h2>source{d} helps you at every step of your Digital Transformation journey</h2>
+        </header>
+        <div class="row">
+            <div class="col-sm-4 mb-5">
+                <div class="circle white-text large-text mb-5">1</div>
+                <h3>Set Targets</h3>
+                <p>What do we want and<br>  what do we get out of it?</p>
+            </div>
+            <div class="col-sm-4 mb-5">
+                <div class="circle white-text large-text mb-5">2</div>
+                <h3>Assess & Plan</h3>
+                <p>Where are we now and<br> what should change?</p>
+            </div>
+            <div class="col-sm-4 mb-5">
+                <div class="circle white-text large-text mb-5">3</div>
+                <h3>Execute & Monitor</h3>
+                <p>Who to involve and<br> How do we measure progress?</p>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section id="visibility" class="visibility pt-5">
```

Comment:
Is this section empty on purpose?
From your screenshot I saw:

![image](https://user-images.githubusercontent.com/2437584/59274937-9a4a8b80-8c5b-11e9-8a3d-25a6ad84b4da.png)

Reply:
That is the old version. In the new one is empty but I kept the block as it adds the curves of the top.

Reply:
It seems that the empty block cause a double wave, what looks strange, doesn't it?

![image](https://user-images.githubusercontent.com/2437584/59420162-ac980700-8dcc-11e9-9da8-133ef351d1b8.png)

Reply:
waves are added by `#journey` and `#how-works`, so the empty `#visibility` container does not seem to be needed.
Could you confirm that you're not seeing double waves in [staging/why-sourced/#visibility](https://landing-staging.srcd.run/why-sourced/#visibility)? Because I do, but I wonder if it could be a problem only in my Chrome.
What about you @vcoisne ?